src/core/services/theming/theming.js

```diff
@@ -190,6 +190,7 @@ var DARK_DEFAULT_HUES = {
   }
 };
 
+// TODO these references no longer exist with the MD spec update of 2018.
```

Comment:
Generally, you can replace all `https://material.google.com/` URLs with `https://material.io/archive/guidelines/`

The new URLs would be:

https://material.io/archive/guidelines/style/color.html#color-usability (new fragment) 
https://material.io/archive/guidelines/style/icons.html#icons-system-icons